src/pages/Landing/index.tsx

```diff
@@ -147,6 +174,12 @@ export default () => {
                 <ColoredTd color={color} dataLabel='isCool'>
                   {isCool ? 'Yup' : 'Totally Not!'}
                 </ColoredTd>
+                <Td key={`kebab-menu-${key}`}>
+                  <ActionsColumn items={actionItems} />
```

Comment:
This is the right direction!  

but you don't need this at all:
```
                  <div>
                    <button onClick={() => handleDelete(name)}>Delete</button>
                  </div>
```

so the problem with the kabab right now, is that the items in it aren't fully correct

so instead of
```
items={actionItems}
```
think about making it call a function:
```
items={actionItems(name)}
```

then its passing the name for each item into a function that is returning the action items specific to that 'name'
right now your action items are referring to a name variable that doesn't exist